data/logos.csv

```diff
@@ -41496,14 +41549,20 @@ ZiboNewsChannel.cn,,,242,150,JPEG,https://www.tvchinese.net/uploads/tv/zbxwpd.jp
 ZiboScienceEducationChannel.cn,,,200,150,JPEG,https://www.tvchinese.net/uploads/tv/zbkjpd.jpg
 ZICOTV.rs,,,470,403,PNG,https://i.imgur.com/8wbTnNW.png
 ZICOTV.rs,,picons,220,132,PNG,https://i.postimg.cc/rySct02p/zicotv.png
-ZiggoSport.nl,,,448,280,PNG,https://i.imgur.com/51PLPwG.png
-ZiggoSport.nl,,picons,220,132,PNG,https://i.postimg.cc/15sS0myr/ziggosport.png
-ZiggoSportDocu.nl,,,47,64,PNG,https://i.imgur.com/lZFxIlk.png
-ZiggoSportGolf.nl,,,47,64,PNG,https://i.imgur.com/Ar4Ipx7.png
-ZiggoSportRacing.nl,,,54,64,PNG,https://i.imgur.com/e8HpSH5.png
-ZiggoSportSelect.nl,,,52,64,PNG,https://i.imgur.com/RqXofNH.png
-ZiggoSportTennis.nl,,,46,60,PNG,https://i.imgur.com/Q2luGei.png
-ZiggoSportVoetbal.nl,,,64,64,PNG,https://i.imgur.com/NtGDkBo.png
+ZiggoSport.nl,,wiki-sm,220,84,PNG,https://upload.wikimedia.org/wikipedia/commons/thumb/0/0e/Ziggo_Sport_logo.svg/220px-Ziggo_Sport_logo.svg.png
+ZiggoSport.nl,,wiki-lg,127,48,SVG,https://upload.wikimedia.org/wikipedia/commons/0/0e/Ziggo_Sport_logo.svg
+ZiggoSport2.nl,,wiki-sm,220,84,PNG,https://upload.wikimedia.org/wikipedia/commons/thumb/0/0e/Ziggo_Sport_logo.svg/220px-Ziggo_Sport_logo.svg.png
+ZiggoSport2.nl,,wiki-lg,127,48,SVG,https://upload.wikimedia.org/wikipedia/commons/0/0e/Ziggo_Sport_logo.svg
+ZiggoSport3.nl,,wiki-sm,220,84,PNG,https://upload.wikimedia.org/wikipedia/commons/thumb/0/0e/Ziggo_Sport_logo.svg/220px-Ziggo_Sport_logo.svg.png
+ZiggoSport3.nl,,wiki-lg,127,48,SVG,https://upload.wikimedia.org/wikipedia/commons/0/0e/Ziggo_Sport_logo.svg
+ZiggoSport4.nl,,wiki-sm,220,84,PNG,https://upload.wikimedia.org/wikipedia/commons/thumb/0/0e/Ziggo_Sport_logo.svg/220px-Ziggo_Sport_logo.svg.png
+ZiggoSport4.nl,,wiki-lg,127,48,SVG,https://upload.wikimedia.org/wikipedia/commons/0/0e/Ziggo_Sport_logo.svg
+ZiggoSport5.nl,,wiki-sm,220,84,PNG,https://upload.wikimedia.org/wikipedia/commons/thumb/0/0e/Ziggo_Sport_logo.svg/220px-Ziggo_Sport_logo.svg.png
+ZiggoSport5.nl,,wiki-lg,127,48,SVG,https://upload.wikimedia.org/wikipedia/commons/0/0e/Ziggo_Sport_logo.svg
+ZiggoSport6.nl,,wiki-sm,220,84,PNG,https://upload.wikimedia.org/wikipedia/commons/thumb/0/0e/Ziggo_Sport_logo.svg/220px-Ziggo_Sport_logo.svg.png
+ZiggoSport6.nl,,wiki-lg,127,48,SVG,https://upload.wikimedia.org/wikipedia/commons/0/0e/Ziggo_Sport_logo.svg
+ZiggoSportExtra.nl,,wiki-sm,220,84,PNG,https://upload.wikimedia.org/wikipedia/commons/thumb/0/0e/Ziggo_Sport_logo.svg/220px-Ziggo_Sport_logo.svg.png
+ZiggoSportExtra.nl,,wiki-lg,127,48,SVG,https://upload.wikimedia.org/wikipedia/commons/0/0e/Ziggo_Sport_logo.svg
```

Comment:
Make a seperate channel instead of renaming. For all renamed channels please mark the old ones  with a closure date.